api/src/core/SceneRunner.cs

```diff
@@ -61,17 +60,13 @@ internal sealed class SceneRunner : ISceneRunner
     private readonly ICollection<Key> keyOnPress = new HashSet<Key>();
     private readonly ICollection<MouseButton> mouseButtonOnPress = new HashSet<MouseButton>();
 
-    public SceneRunner(string resourcePath, bool autoFree = false, bool verbose = false)
+    public SceneRunner(Node currentScene, bool autoFree = false, bool verbose = false)
```

Comment:
Do not change the original constructor,  instead add a second constructor where accepts a PackedScene as argument